pkg/admin/admin.go

```diff
@@ -31,15 +34,26 @@ func StartServer(addr string) {
 }
 
 func (h *handler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+	debugPathPrefix := "/debug/pprof/"
 	switch req.URL.Path {
 	case "/metrics":
 		h.promHandler.ServeHTTP(w, req)
 	case "/ping":
 		h.servePing(w)
 	case "/ready":
 		h.serveReady(w)
+	case fmt.Sprintf("%scmdline", debugPathPrefix):
+		pprof.Cmdline(w, req)
+	case fmt.Sprintf("%sprofile", debugPathPrefix):
+		pprof.Profile(w, req)
+	case fmt.Sprintf("%strace", debugPathPrefix):
```

Comment:
curious: why not implement `/debug/pprof/symbol` for completeness?

Reply:
Ah good call. Didn't realize I left that one out 👍